bot/utils/blurple_formatter.py

```diff
@@ -701,22 +807,24 @@ def unparse(node: ast.AST, nl_able: bool = False) -> str:
         target, iter_, ifs, is_async = node.target, node.iter, node.ifs, node.is_async
 
         return (
-            f"async{space(1)}" if is_async else ""
-            + f"for{space(1)}{unparse(target)}{space(1)}in{space(1)}{unparse(iter_)}"
-            + (f"{space(1)}" if ifs else "")
-            + f"{space(1)}".join(f"if{space(1)}{unparse(if_)}" for if_ in ifs)
+            f"async{space()}" if is_async else ""
+            + f"for{space()}{unparse(target)}{space()}in{space()}{unparse(iter_)}"
+            + (f"{space()}" if ifs else "")
+            + f"{space()}".join(f"if{space()}{unparse(if_)}" for if_ in ifs)
         )
     if isinstance(node, ast.withitem):
         context_expr, optional_vars = node.context_expr, node.optional_vars
 
         return f"{unparse(context_expr)}" + (
-            f"{space(1)}as{space(1)}{unparse(optional_vars)}" if optional_vars else ""
+            f"{space()}as{space()}{unparse(optional_vars)}" if optional_vars else ""
         )
 
     return ast.unparse(node)
 
 
 def blurplify(src: str) -> str:
     """Format the given source code in accordance with PEP 9001."""
+    global source # use a global variable only to get the original source part of constants
     src_ast = ast.parse(src)
+    source = src
```

Comment:
Can you move this line to be right below `global source`? It is a bit confusing having the `ast.parse()` call between them.

Reply:
Doing it.